src/markup.c

```diff
@@ -66,8 +66,10 @@ char *markup_strip(char *str)
         /* strip all tags */
         string_strip_delimited(str, '<', '>');
 
-        /* unquote the remainder */
+        /* We want to quote; but unquote first to avoid
+         * double-escaping anything already quoted. */
         str = markup_unquote(str);
+        str = markup_quote(str);
```

Comment:
I was confused by your statement, then I realized that I misunderstood the code in `r_create_layout_from_notification`.